Sources/Client/TezosClient.swift

```diff
@@ -220,15 +220,31 @@ public class TezosClient {
 
   /**
    * Originate a new account from the given account.
+   *
+   * @param managerAddress The address which will manage the originated contract.
+   * @param keys The keys to use to sign the operation for the address.
+   * @param contractCode Optional code to associate with the originated contract.
+   * @param completion A completion block which will be called with a string representing the
+   *        transaction ID hash if the operation was successful.
    */
-  public func originateAccount(managerAddress: String, keys: Keys, completion: @escaping (String?, Error?) -> Void) {
-    let originateAccountOperation = OriginateAccountOperation(address: managerAddress)
+  public func originateAccount(managerAddress: String, keys: Keys, contractCode: ContractCode? = nil, completion: @escaping (String?, Error?) -> Void) {
```

Comment:
Have you setup Swiftlint yet?  Try to limit lines to 120 chars so they are easier to review

Reply:
I use swiftformat. The config is  here: https://github.com/keefertaylor/TezosKit/blob/master/.swiftformat

I'll look into whether it supports line length customization.

Reply:
Swiftlint catches code smells that the formatter doesn't.  I use both on all my projects.  See kratos for the .swiftlint.yml